lib/exception_handling.rb

```diff
@@ -202,6 +204,8 @@ def log_error(exception_or_string, exception_context = '', controller = nil, tre
 
         write_exception_to_log(ex, exception_context, timestamp)
 
+        send_exception_to_honeybadger(ex)
```

Comment:
Worth bringing up in the demo, we but might want to include exception_context and timestamp if we find those useful.